R/multitables.R

```diff
@@ -26,6 +26,41 @@ setMethod("[[", c("MultitableCatalog", "numeric"), function (x, i, ...) {
     getEntity(x, i, Multitable, ...)
 })
 
+#' @rdname catalog-extract
+#' @export
+setMethod("[[<-", c("MultitableCatalog", "character", "missing", "formula"),
+          function (x, i, j, value) {
+              stopifnot(length(i) == 1)
```

Comment:
I'm generally not a fan of keeping dead code around, but it's tested and it's not big, so we can keep it if you want. We will probably be in this code again soon and can reevaluate then.